src/makeInfinityAdder.js

```diff
@@ -5,6 +5,30 @@
  */
 function makeInfinityAdder() {
   // write code here
+  let total = 0;
+  return (a) => {
+    total = a;
```

Comment:
The assignment `total = a;` will overwrite the running total with the current argument, which is not the intended behavior. The `adder` function should accumulate the sum of all arguments until it encounters a call without parameters.